FogCreek/Glitch-Community
# Sumptuous rainbow: styled components SSR, render '→' as icon

## Links
* Remix link: http://sumptuous-rainbow.glitch.me
* Issue-tracker link: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/111

## Changes:
* add babel plugin, SSR for styled components 
* replace arrow unicode with icon for consistent alignment

## How To Test:
* visit http://sumptuous-rainbow.glitch.me
* there should be no 'flash of unstyled content' while shared components styles load
* arrows in buttons should be consistently rendered & aligned across browsers & platforms
